byterun/caml/minor_gc.h

```diff
@@ -46,7 +46,7 @@ struct caml_custom_elt {
 struct caml_custom_table CAML_TABLE_STRUCT(struct caml_custom_elt);
 
 struct caml_minor_tables {
-  struct caml_ref_table major_ref, minor_ref;
+  struct caml_ref_table major_ref, major_ref_rewrites, minor_ref;
```

Comment:
Since this is only used temporarily inside `caml_empty_minor_heap_domain`, it would probably be better to have it as a local variable in that function.

Reply:
agree, fixed!